Core/Src/datalogging/odometer_calc.c

```diff
@@ -18,63 +18,68 @@
 #include "util/global_instances.h"
 #include "util/util.h"
 
-static inline uint32_t calculate_distance(uint32_t speed_avg);
-static void log_distance(uint32_t total_dist, uint32_t partial_dist);
+// static inline uint32_t calculate_distance(uint32_t speed_avg);
+// static void log_distance(uint32_t total_dist, uint32_t partial_dist);
 
 void odometer_calc() {
 
-    uint32_t partial_dist_traveled = 0;
-    odometer_message_t total_dist_traveled;
-    uint16_t odometer_speed_data = 0;
-
-    // Read distance from flash just once
-    Flash_Read_Data(ODOMETER_DATA_FLASH_ADDR, &(total_dist_traveled), WORDS_READ_ONE);
-
-    // In case the memory position has never been used/saved
-    if ((total_dist_traveled == 0x00000000) || (total_dist_traveled == 0xFFFFFFFF)) {
-        total_dist_traveled = 0;
-    }
-    total_dist_traveled = m_to_cm(total_dist_traveled);
-    osMessageQueuePutOverwrite(q_odometer_calc_save_messageHandle, &total_dist_traveled,
-                               0);
+    //    uint32_t partial_dist_traveled = 0;
+    //    odometer_message_t total_dist_traveled;
+    //    uint16_t odometer_speed_data = 0;
+    //
+    //    // Read distance from flash just once
+    //    Flash_Read_Data(ODOMETER_DATA_FLASH_ADDR, &(total_dist_traveled),
+    //    WORDS_READ_ONE);
+    //
+    //    // In case the memory position has never been used/saved
+    //    if ((total_dist_traveled == 0x00000000) || (total_dist_traveled == 0xFFFFFFFF))
+    //    {
+    //        total_dist_traveled = 0;
+    //    }
+    //    total_dist_traveled = m_to_cm(total_dist_traveled);
+    //    osMessageQueuePutOverwrite(q_odometer_calc_save_messageHandle,
```

Comment:
Pode voltar com esse arquivo. O save poderia voltar, mas deixa sem por enquanto. Pelo menos aqui vai poder datalogar a distancia parcial

Reply:
To indo sem sensor de velocidade. Funciona em bancada, mas não ta pegando a roda fônica. A fixação não ta legal. Pode deixar sem o calc mesmo?

Reply:
manda le do sensor traseiro, legal ter esse dado

Reply:
Ok